templates/cli/lib/commands/generic.js.twig

```diff
@@ -68,6 +69,13 @@ const client = new Command("client")
             try {
                 let url = new URL(endpoint);
                 if (url.protocol === "http:" || url.protocol === "https:") {
+                    let apClient = new Client().setEndpoint(endpoint);
+                    let response = await apClient.call('GET', '/health/version');
+                    if(response.version) {
+                        globalConfig.setEndpoint(endpoint);
+                    } else {
+                        throw new Error("Invalid endpoint or your Appwrite server is not running as expected.");
```

Comment:
```suggestion
                        throw new Error();
```
Just throw the error and add the message `Invalid endpoint or your Appwrite server is not running as expected.` 
here 
https://github.com/appwrite/sdk-generator/blob/fbd6effddaaba3f9ef6a6725b3df648436fe8212/templates/cli/lib/commands/generic.js.twig#L76